docs/concepts/functions-overview.md

```diff
@@ -15,6 +15,7 @@ Batch runtimes:
 * **local** - execute a Python or shell program 
 * **{ref}`job <job-function>`** - run the code in a Kubernetes Pod
 * **{ref}`dask <dask-overview>`** - run the code as a Dask Distributed job (over Kubernetes)
+* **{ref}`databricks <databricks>`** - run code on Databricks cluster (python scripts, Spark etc)
```

Comment:
```suggestion
* **{ref}`databricks <databricks>`** - Run code on a Databricks cluster as a Python script.
```

Reply:
@assaf758 - the current text is yours. Need you two to agree on this. Thanks

Reply:
ok np